### ISSUE TITLE
Turn off legend clickability

### ISSUE THREAD

**Issue by brockfanning** [open-sdg/open-sdg#909]
Right now there is one way to add datasets to a chart: checking the checkboxes in the sidebar. However there are *two* ways to remove datasets from the chart: unchecking the checkboxes in the sidebar, and clicking the items in the legend. Unfortunately these two ways are not synchronized - ie, when you click the item in the legend, it does _not_ uncheck the checkbox in the sidebar. This can cause confusion when a previously-hidden dataset (hidden via the legend) suddenly appears because another checkbox was checked.

All things considered it would be better if the legend were simply a legend, and not a clickable widget for hiding datasets. Let's make this changed, but allow for an easy way to implementations to get back the clickability if necessary.

**Comment by LucyGwilliamAdmin** [comment]
We've decided to remove legend but leaving this feedback from DAC retest in case needed in future:

**URL:** http://sdgdev-813006012.eu-west-1.elb.amazonaws.com/dac2020/3-a-1/

**Screen shot:**
![image](https://user-images.githubusercontent.com/43404358/94578878-1f521d80-0270-11eb-836c-4ed1d8091ccb.png)

When the United Kingdom legend button is activated, the state updates to ‘pressed’, but the label still reads ‘Press to hide’ rather than ‘Press to show’.

**Current Code Ref(s):** #legend > li.notshown > button

```
<button aria-pressed="true" aria-label="Press to hide the data set United Kingdom"> <span class="swatch" style="background-color: #777777"></span> United Kingdom </button>
```

**Solution:**
Please ensure that labels and instructions correctly describe the component.

**Comment by brockfanning** [comment]
Fixed by #914

**Comment by brockfanning** [comment]
Also just to follow up on backwards compatibility - as you can gather from the PR that fixed this (#914) there is not an easy way for implementations to add back this legend click behavior. We do not consider this a breaking change, because it is a user-interface improvement.
